zenodo/modules/records/data/objecttypes.json

```diff
@@ -194,6 +194,25 @@
   },
   "csl": "graphic"
 },
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "workflow",
+  "id": "https://zenodo.org/objecttypes/workflow#",
+  "title": {
+    "en": "Workflow"
+  },
+  "title_plural": {
+    "en": "Workflows"
+  },
+  "datacite": {"general": "Workflow"},
+  "eurepo": "info:eu-repo/semantics/other",
+  "schema.org": "https://schema.org/CreativeWork",
```

Comment:
We leave it like this for now, but we should look into future changes in schema.org. Here's also an interesting issue discussing metadata mappings: https://github.com/seek4science/seek/issues/636